project_testing/measures/SimulationOutputReport/measure.rb

```diff
@@ -90,49 +98,14 @@ def outputs
       "hvac_heating_capacity_w",
       "hvac_heating_supp_capacity_w",
       "upgrade_name",
-      "upgrade_cost_usd",
-      "upgrade_option_01_cost_usd",
-      "upgrade_option_01_lifetime_yrs",
-      "upgrade_option_02_cost_usd",
-      "upgrade_option_02_lifetime_yrs",
-      "upgrade_option_03_cost_usd",
-      "upgrade_option_03_lifetime_yrs",
-      "upgrade_option_04_cost_usd",
-      "upgrade_option_04_lifetime_yrs",
-      "upgrade_option_05_cost_usd",
-      "upgrade_option_05_lifetime_yrs",
-      "upgrade_option_06_cost_usd",
-      "upgrade_option_06_lifetime_yrs",
-      "upgrade_option_07_cost_usd",
-      "upgrade_option_07_lifetime_yrs",
-      "upgrade_option_08_cost_usd",
-      "upgrade_option_08_lifetime_yrs",
-      "upgrade_option_09_cost_usd",
-      "upgrade_option_09_lifetime_yrs",
-      "upgrade_option_10_cost_usd",
-      "upgrade_option_10_lifetime_yrs",
-      "upgrade_option_11_cost_usd",
-      "upgrade_option_11_lifetime_yrs",
-      "upgrade_option_12_cost_usd",
-      "upgrade_option_12_lifetime_yrs",
-      "upgrade_option_13_cost_usd",
-      "upgrade_option_13_lifetime_yrs",
-      "upgrade_option_14_cost_usd",
-      "upgrade_option_14_lifetime_yrs",
-      "upgrade_option_15_cost_usd",
-      "upgrade_option_15_lifetime_yrs",
-      "upgrade_option_16_cost_usd",
-      "upgrade_option_16_lifetime_yrs",
-      "upgrade_option_17_cost_usd",
-      "upgrade_option_17_lifetime_yrs",
-      "upgrade_option_18_cost_usd",
-      "upgrade_option_18_lifetime_yrs",
-      "upgrade_option_19_cost_usd",
-      "upgrade_option_19_lifetime_yrs",
-      "upgrade_option_20_cost_usd",
-      "upgrade_option_20_lifetime_yrs",
-      "weight"
+      "upgrade_cost_usd"
     ]
+    for option_num in 1..num_options
+      buildstock_outputs << "upgrade_option_%02d_cost_usd" % option_num
+      buildstock_outputs << "upgrade_option_%02d_lifetime_yrs" % option_num
+    end
```

Comment:
👍

Reply:
I assume it's not a big deal that you're changing the order of the outputs (and, presumably, results.csv columns)?

Reply:
How is this changing the order of the outputs?

Reply:
It looks like "upgrade_cost_usd" used to be last in the list (after "weight"), now it's before all the "upgrade_option_foo"?

Reply:
Oh wait, I think I'm reading the changes wrong. Maybe it is the same.

Reply:
Nevermind, it looks good.

Reply:
Aren't they before? https://github.com/NREL/OpenStudio-BuildStock/blob/master/measures/SimulationOutputReport/measure.rb#L92-L93